labeling_guide/ReadMe.md

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
We should move everything about labeling in this document, put it in the top-level directory calling it "LABELING.MD", and referencing it in README.md.